superdesk/errors.py

```diff
@@ -558,5 +558,25 @@ def httpPushError(cls, exception=None, destination=None):
         return PublishHTTPPushError(14000, exception, destination)
 
 
+class PublishHTTPPushClientError(PublishHTTPPushError):
+    _codes = {
+        14001: "HTTP push publish client error",
+    }
+
+    @classmethod
+    def httpPushError(cls, exception=None, destination=None):
+        return PublishHTTPPushClientError(14001, exception, destination)
+
+
+class PublishHTTPPushServerError(PublishHTTPPushError):
+    _codes = {
+        14002: "HTTP push publish server error",
+    }
+
+    @classmethod
```

Comment:
now I see why you said that those static methods are weird. imo the idea was to have them on the base class, so you can raise more specific errors without importing them all, to have it on those specific error classes is indeed useless..